code/modules/surgery/organs/augments_arms.dm

```diff
@@ -604,3 +604,75 @@
 	if(emp_proof)
 		return
 	muscle_implant.emp_act(severity, owner)
+
+/obj/item/organ/internal/cyberimp/arm/power_cord_cell
+	name = "Cell-compatible power adapter implant"
+	desc = "This is a recently developed implant for installation inside IPCs in order to allow them to easily collect energy from cells"
+	origin_tech = "materials=3;engineering=4;biotech=3"
```

Comment:
One more thing. you've given it quite a high engineering origin but there's no requirement for tech on the mechfab side of things, i'd suggest making it atleast the same as the apc one.

Reply:
> One more thing. you've given it quite a high engineering origin but there's no requirement for tech on the mechfab side of things, i'd suggest making it atleast the same as the apc one.

![image](https://github.com/ParadiseSS13/Paradise/assets/58084138/56ef7c47-9792-4f4a-92f0-afafb465f983)
Strange. I have different techs

Reply:
Dun. Now original  materials=3;biotech=2;powerstorage=3